Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/README.md

```diff
@@ -97,7 +97,7 @@ Displays the available feeds from the MITRE taxii service.
 
 ##### Base Command
 
-`mitre-showfeeds`
+`mitre-show-feeds`
```

Comment:
Is this just a readme fix?

Reply:
yes, the command in the .yml file is `mitre-show-feeds` and in the README it was mitre-showfeeds by mistake I guess. The validation raised an error regarding the name differences.